backend/src/main/java/hu/flowacademy/vajdasagbrand/entity/UserDTO.java

```diff
@@ -0,0 +1,17 @@
+package hu.flowacademy.vajdasagbrand.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDTO {
+    private String fullName;
```

Comment:
Rakj be egy-egy `@JsonProperty`-t minden olyan mezőre ami camelCase és legyen a json alakjuk snake_case